python/sglang/srt/layers/quantization/modelopt_quant.py

```diff
@@ -49,20 +49,21 @@
     )
     from sglang.srt.single_batch_overlap import DownGemmOverlapArgs
 
+fp4_gemm = None
 if is_cuda():
+    from sgl_kernel import cutlass_scaled_fp4_mm as fp4_gemm
     from sgl_kernel import scaled_fp4_quant
 
+    enable_flashinfer_fp4_gemm = False
+
 try:
-    from flashinfer import mm_fp4 as fp4_gemm
     from flashinfer import reorder_rows_for_gated_act_gemm, shuffle_matrix_sf_a
 
-    enable_flashinfer_fp4_gemm = True
+    if not is_cuda():
```

Comment:
When is_cuda is false, we should not use flashinfer fp4 gemm here.

Reply:
I think we could merge this first. After flashinfer fixes this, we could revert this pr directly. wdyt?